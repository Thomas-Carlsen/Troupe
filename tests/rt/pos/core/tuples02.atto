let val x = (10,20) 
in x 
end 
