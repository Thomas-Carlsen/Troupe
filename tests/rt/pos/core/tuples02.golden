2019-03-07T12:08:15.833Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: (10@{}%{}, 20@{}%{})@{}%{}
